### PR TITLE
Winging It 6

### PR DESCRIPTION
**Pull request by stacyk**
## Description
Winging it 6!

## Related Issue(s)
#507 

## Steps to test/reproduce
Check out [homepage](https://deploy-preview-565--oddleventy.netlify.app/#featured-posts) posts, then [posts page](https://deploy-preview-565--oddleventy.netlify.app/blog/), then [post single for winging it 6](https://deploy-preview-565--oddleventy.netlify.app/2024/04/18/winging-it-06/)

## Show me
posts-page
post-single

### CODE REVIEW

File: content/blog/wingingit/winging-it-06.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- When and How to use Proxies
+- When to avoid using Proxies
+- Proxies in the real world
+
+## Links:
+
+- [Proxy on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
+- [Signals Proposal](https://github.com/proposal-signals/proposal-signals)
+- [Proxy is What’s in Store](https://www.oddbird.net/2024/01/12/proxy-store/)
+- [Sass](https://sass-lang.com/)
```
Comment (jamesnw) [inline content/blog/wingingit/winging-it-06.md:49]:
```suggestion
- [Sass Playground](https://sass-lang.com/playground)
```

File: content/blog/wingingit/winging-it-06.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+
+- What is state?
+- State management solutions
+- What are Proxies?
+- When and How to use Proxies
```
Comment (jamesnw) [inline content/blog/wingingit/winging-it-06.md:40]:
```suggestion
- When and How to use Proxies to manage state
```

File: content/blog/wingingit/winging-it-06.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- When to avoid using Proxies
```
Comment (jamesnw) [inline content/blog/wingingit/winging-it-06.md:41]:
```suggestion
- When to avoid using Proxies for state management
```

For SEO? 😆 😭

**Review by jamesnw — APPROVED** [review]
Looks great!

File: content/blog/wingingit/winging-it-06.md (hunk 1 above)
Comment (stacyk) [reply content/blog/wingingit/winging-it-06.md:49]: @jamesnw I added [this link](https://sass-lang.com/playground/#eJxdj9FqwjAUhu/zFD9OVgULyhAhvfFV0ubYhiY5JYk4J333xdluZTch5zv5v58cDus6kOoHNj5FiY1AEZ2ythxUS4XExz6Q22XqSJurm/HxOGGrQkszPb3othJCnEk1HRb2HdaxCUS+jOaLYPxyGfEQwPlZovB6BuU1Ns748mZ06iSKt8fSMBbbnxBQs75PV+BdypouHOiXAA37RD5JrLLir3TEwv5Pvqqm9CjmcxRj/tezbFlRq6ZvA1+9Lhu2HCQSO5U456f51plEedQmDlbdJWrLTZ/BwNEkw17iYj5JZ5J4kNhXYvwGtNZ+7g==) to some sample code already in place

**Pull request merged**